code/modules/mob/living/blood.dm

```diff
@@ -29,12 +29,8 @@
 				b_volume = 0
 			else if(chem_effect_flags & CHEM_EFFECT_ORGAN_STASIS)
 				b_volume *= 1
-			else if(heart.damage > 1 && heart.damage < heart.min_bruised_damage)
-				b_volume *= 0.8
-			else if(heart.damage >= heart.min_bruised_damage && heart.damage < heart.min_broken_damage)
-				b_volume *= 0.6
-			else if(heart.damage >= heart.min_broken_damage && heart.damage < INFINITY)
-				b_volume *= 0.3
+			else if(heart.damage >= heart.min_bruised_damage)
```

Comment:
This should use organ status, something like heart.organ_status >= ORGAN_BRUISED